test/test_bucket.py

```diff
@@ -3,66 +3,63 @@
 from uri.bucket import Bucket
 
 EXAMPLES = [
-		# String,       Arguments,         Name,    Value,      Valid
-		
-		( '',           ('', ),            None,    '',         True ),
-		( 'foo',        ('foo', ),         None,    'foo',      True ),
-		( 'foo',        (None, 'foo'),     None,    'foo',      True ),
-		( 'foo=bar',    ('foo', 'bar'),    'foo',   'bar',      True ),
-		
-		( '=foo=bar',   ('=foo=bar', ),    '',      'foo=bar',   False ),
-		( '=foo=bar',   ('=foo', 'bar'),   '=foo',  'bar',       False ),
-		( '=foo=bar',   ('', 'foo=bar'),   '',      'foo=bar',   False ),
-		( 'foo=bar=',   ('foo', 'bar='),   'foo',   'bar=',      False ),
-		( 'foo==bar=',  ('foo=', 'bar='),  'foo=',  'bar=',      False ),
-		( 'foo==bar=',  ('foo==bar=', ),   'foo',   '=bar=',     False ),
-		( '=foo=bar=',  ('=foo=bar=', ),   '',      'foo=bar=',  False ),
-		
-	]
+    # String,       Arguments,         Name,    Value,      Valid
+    ("", ("",), None, "", True),
+    ("foo", ("foo",), None, "foo", True),
+    ("foo", (None, "foo"), None, "foo", True),
+    ("foo=bar", ("foo", "bar"), "foo", "bar", True),
+    ("=foo=bar", ("=foo=bar",), "", "foo=bar", False),
+    ("=foo=bar", ("=foo", "bar"), "=foo", "bar", False),
+    ("=foo=bar", ("", "foo=bar"), "", "foo=bar", False),
+    ("foo=bar=", ("foo", "bar="), "foo", "bar=", False),
+    ("foo==bar=", ("foo=", "bar="), "foo=", "bar=", False),
+    ("foo==bar=", ("foo==bar=",), "foo", "=bar=", False),
+    ("=foo=bar=", ("=foo=bar=",), "", "foo=bar=", False),
+]
```

Comment:
I hope you're using automated code formatting for this, and possibly the prior hunk. This had a tabular alignment for a reason.